google/cloud/aiplatform/initializer.py

```diff
@@ -42,6 +41,10 @@
 class _Config:
     """Stores common parameters and options for API calls."""
 
+    # Class attribute to set user_agent header used for metrics reporting.
+    user_agent_product = "model-builder"
```

Comment:
I have a preference for storing this under `constants.py`. So setting is a bit more direct and aligned with setting the `API_BASE_PATH`. ie:

```aiplatform.constants.USER_AGENT_PRODUCT = "other-product"```

Reply:
Done, thats a good call, thank you!